sympy/matrices/tests/test_matrices.py

```diff
@@ -1332,11 +1332,11 @@ def test_vech():
 
     m = Matrix([[1, x*(x + y)], [y*x + x**2, 1]])
     m_vech = m.vech(diagonal=False)
-    assert m_vech[0] == x*(x + y)
+    assert m_vech[0].equals(x*(x + y))
 
     m = Matrix([[1, x*(x + y)], [y*x, 1]])
     m_vech = m.vech(diagonal=False, check_symmetry=False)
-    assert m_vech[0] == y*x
+    assert m_vech[0].equals(y*x)
```

Comment:
I'm not exactly sure how to go about that deprecation.  It seems like the tests should have used `.equals` from the beginning (they aren't testing simplify).  It's the lines above this (the `assert m_vech[0].equals(x*(x + y))`) where the test will now fail without `.equals` because I followed what the docstring said the functions behaviour was...